Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rust-g"
 edition = "2021"
-version = "3.0.0"
+version = "3.1.0"
```

Comment:
we generally don't update versions in feature PRs

Reply:
Ok